inf-clojure.el

```diff
@@ -725,19 +731,24 @@ The value is nil if it can't find one."
 (defun inf-clojure-show-var-documentation (var)
   "Send a form to the inferior Clojure to give documentation for VAR.
 See function `inf-clojure-var-doc-form'."
-  (interactive (inf-clojure-symprompt "Var doc" (inf-clojure-var-at-pt)))
-  (comint-proc-query (inf-clojure-proc) (format (inf-clojure-var-doc-form) var)))
+  (interactive "P")
+  (let ((var (if (inf-clojure--prompt-on-commands-p var)
```

Comment:
You're killing me, buddy. :-) `var` is a prefix arg at this point, but after the `if` returns it's overridden - it becomes the symbol at point. I assumed this was apparent. I find this extremely confusing, so I'd change the parameter name to something else, the `let`-binding is fine. Hopefully, I managed to make myself clear this time around.